index.php

```diff
@@ -0,0 +1,270 @@
+
+        <section id="slider">
+            <div>
+                <img src="public/images/slider.png" alt="Background coucher de soleil">
+            </div>
+            <div class="background"></div>
+            <div class="pres row align-items-center headline">
+                <h1>
+                    <span class="tagline">Frédéric Vanmarcke</span>
+                    <br />
+                    <span class="int tagline">Développeur - Intégrateur Web & Web Mobile à Rambouillet</span>
+                   
+                </h1>
+            </div>
+            <a id="toDown" title="Go to down" href="#me" class="fa fa-chevron-down punchline">
+            </a>
+        </section>
+        <!-- SLIDER END ***************************************** -->
+        
+        <?php
+        // ABOUT ME *****************************************
+
+         include 'public/view/aboutMe.twig';
+
+        //  TRAINING *****************************************
+        
+         include 'public/view/training.twig';
+
+        //  EXPERIENCE *****************************************
+       
+         include 'public/view/experience.twig';
+
+        //  SKILL  *****************************************
+    
+         include 'public/view/skill.twig';
+
+        //  PORTFOLIO  *****************************************
+     
+         include 'public/view/portFolio.twig';
+
+        //  HOBBIES  *****************************************
+        
+         include 'public/view/hobbies.twig';
+        ?>
+        <!-- CONTACT  ***************************************** -->
+       
+        <section id="contact">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h1 class="titreBorder">Mes réseaux</h1>
+                </div>
+                <div class="col-lg-12">
+                    <ul class="networks">
+                        <li>
+                            <a href="https://linkedin.com/in/frédéric-vanmarcke/" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
+                        </li>
+
+                        <li>
+                            <a href="https://github.com/vanmarcke/" target="_blank" rel="noopener" class="fa fa-github"></a>
+                        </li>
+                        <li>
+                            <a href="https://www.pinterest.fr/fvanmarcke72/" target="_blank" rel="noopener" class="fa fa-pinterest"></a>
+                        </li>
+                        <li>
+                            <a href="https://www.facebook.com/guedin78/" target="_blank" rel="noopener" class="fa fa-facebook"></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            
+            <div class="col-lg-12">
+                <h1 class="titreBorder">Me contacter</h1>
+            </div>
+            <div class="col-lg-8 mx-auto">
+
+                <form method="POST" action="EmailProcessing.php" id="contactForm">
```

Comment:
Ton script EmailPrcessing.php ne devrait pas être à la racine du projet.
https://openclassrooms.com/fr/courses/4670706-adoptez-une-architecture-mvc-en-php/4682551-organiser-en-dossiers

Reply:
Ce fichier sera déplacé dans le dossier controlleur
lors de l'issue "Add file required for routing"